pkg/apis/application/v1alpha1/types.go

```diff
@@ -384,17 +384,19 @@ type ApplicationSourceHelm struct {
 	IgnoreMissingValueFiles bool `json:"ignoreMissingValueFiles,omitempty" protobuf:"bytes,8,opt,name=ignoreMissingValueFiles"`
 	// SkipCrds skips custom resource definition installation step (Helm's --skip-crds)
 	SkipCrds bool `json:"skipCrds,omitempty" protobuf:"bytes,9,opt,name=skipCrds"`
+	// SkipTests skips test manifest installation step (Helm's --skip-tests). By default, Argo CD assumes it is set to true.
+	SkipTests bool `json:"skipTests,omitempty" protobuf:"bytes,10,opt,name=skipTests"`
 	// ValuesObject specifies Helm values to be passed to helm template, defined as a map. This takes precedence over Values.
 	// +kubebuilder:pruning:PreserveUnknownFields
-	ValuesObject *runtime.RawExtension `json:"valuesObject,omitempty" protobuf:"bytes,10,opt,name=valuesObject"`
+	ValuesObject *runtime.RawExtension `json:"valuesObject,omitempty" protobuf:"bytes,11,opt,name=valuesObject"`
```

Comment:
You cannot do that with protobuf. bytes represent the index of fields and must be preserved for backward compatibility. It is better to add the new fields at the end of the object, with the bytes=14.

See https://protobuf.dev/programming-guides/proto3/#assigning